packages/bpk-docs/src/pages/HomePage/HomePage.js

```diff
@@ -186,7 +186,7 @@ const HomePage = (props, context) => {
 };
 
 HomePage.contextTypes = {
-  theme: PropTypes.object,
+  theme: PropTypes.object, // eslint-disable-line react/forbid-prop-types
```

Comment:
How come we didn't have to opt out here before? Seems like it should have already been causing an error.